app/ui/tasks/card-delete-component.tsx

```diff
@@ -0,0 +1,24 @@
+'use client';
+import { MenuItem } from '@chakra-ui/react';
+import React from 'react';
+import { useRouter } from 'next/navigation';
+import { deleteTask } from '@/app/seed/route';
+
+interface ICardDeleteComponent {
```

Comment:
There is no need to add `I` prefix for interfaces. It is obvious that it is interface if we check where it is defined 😄

Reply:
Usually, we call such interfaces something like `CardDeleteComponentProps`